lib/sidekiq_publisher/database_connection.rb

```diff
@@ -2,8 +2,13 @@
 
 module SidekiqPublisher
   module DatabaseConnection
+    # TODO: this is deprecated and should not be used outside if this class
```

Comment:
I was assuming that removing this method from a public module would be a breaking change?  Should I remove the method and bump the major version instead?

Reply:
Hmm, it is confusing that it's a public method. Is it intended to be used outside of the gem? I couldn't find any other application outside of the gem itself using it. I would normally only consider a breaking change if it requires code to change in applications that have it as a dependency. If it is meant to be used externally, though, I would remove it and make this a major change. Otherwise, I'm okay with a minor bump.

Reply:
I do not think it is meant to be used outside the gem, but the fact this gem is public means I am more inclined to do the right thing and major bump.

Reply:
Oop, I stand [corrected](https://github.com/search?q=org%3Aezcater+.transaction_open%3F&type=code) and searched incorrectly the first time. Yeah I would make this a breaking change with a major bump if you want to remove this method.

Reply:
Yeah I think that's a good point, I think to follow general best practices, if we want to remove the module, we can make this a major bump.